server/lib/config.js

```diff
@@ -23,6 +24,10 @@ const nodeEnv = configureNodeEnv();
 console.log(`Config is using nodeEnv=${nodeEnv}`);
 const configDir = path.join(__dirname, '../../config');
 
+// Setup nconf to use (in-order):
+//   1. Command-line arguments (argv)
+//   2. Environment variables (env)
+//   3. `config/config-{env}.json` files
```

Comment:
Based off of https://github.com/indexzero/nconf/blob/master/README.md?plain=1#L15-L19